lib/main.coffee

```diff
@@ -8,6 +8,9 @@ module.exports =
       scopes:
         '.source.jade':
           default: false
+    removeTrailingWhitespaceAsYouType:
```

Comment:
I'd be open to any name, although the association to `removeTrailingWhitespace` is useful, as it doesn't strip whitespace if that isn't enabled, regardless of this setting.